pombola/core/views.py

```diff
@@ -486,6 +486,8 @@ def position(request, pt_slug, ok_slug=None, o_slug=None):
         'object':     title,
         'page_title': page_title,
         'order':      request.GET.get('order'),
+        'org_kind':   organisation_kind,
```

Comment:
`organisation_kind` is already added to the template in the code a bit further down (although I should have pointed out when reviewing that earlier addition that it should just be included in the dictionary literal, as you've done. oops.)

I'd be tempted to make a commit that moves the setting of `organisation_kind` into the dictionary literal first, then a variant of this commit that calls it `organisation_kind`. (Or have another commit to rename `organisation_kind` to `org_kind` - I don't mind about the naming here.)

BTW, we're deliberately not lining up dictionary values in literals in newly introduced code, since it's a PEP8 violation, in case keeping a minimal diff was part of the motivation for the shorter names.